_pages/infrastructure/monitoring.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Monitoring
+parent: Infrastructure
+---
+
+There are a couple kinds of monitoring that you will need to have in place for any application: [errors](#errors) and [downtime](#downtime).
+
+Monitoring is only useful if the relevant people are alerted when something goes wrong, and then only if those individuals
```

Comment:
Maybe mention something here about getting alerts on mobile devices in addition to slack since that proved helpful @yozlet?